appserver/tests/payara-samples/pom.xml

```diff
@@ -53,24 +52,35 @@
         <module>classpath</module>
     </modules>
 
-    <dependencies>
-        <!-- DO NOT ADD DEPENDENCIES -->
-    </dependencies>
```

Comment:
This served as warning, that adding dependency here will break classpath module, and I think it should remain here.

Reply:
Warning not needed, there is maven enforcer plugin to detect it.
Generally there should not be any implicit dependencies, now remained junit, hamcrest and com.sun:tools-jar.